packages/react-sdk/src/components/index.ts

```diff
@@ -96,3 +151,9 @@ export const getComponent = <Name extends ComponentName>(
 export const registerComponents = (components: RegisteredComponents) => {
   registeredComponents = components;
 };
+
+export const registerComponentsMeta = (
+  componentsMeta: RegisteredComponentsMeta
+) => {
+  registeredComponentsMeta = componentsMeta;
+};
```

Comment:
registry implementation should probably be a separate module, so that there is no mixture of definition and registry itself. In the end registry will be completely reusable for all sdk's, but definitions code is specific to each sdk.